src/tiered/tiered_cursor.c

```diff
@@ -418,20 +417,20 @@ __curtiered_next(WT_CURSOR *cursor)
     WT_ERR(__curtiered_enter(curtiered, false));
 
     /* If we aren't positioned for a forward scan, get started. */
-    if (curtiered->current == NULL || !F_ISSET(curtiered, WT_CLSM_ITERATE_NEXT)) {
+    if (curtiered->current == NULL || !F_ISSET(curtiered, WT_CURTIERED_ITERATE_NEXT)) {
         WT_FORALL_CURSORS(curtiered, c, i)
         {
             if (!F_ISSET(cursor, WT_CURSTD_KEY_SET)) {
                 WT_ERR(c->reset(c));
                 ret = c->next(c);
             } else if (c != curtiered->current &&
-              (ret = __curtiered_position_chunk(curtiered, c, true, &cmp)) == 0 && cmp == 0 &&
+              (ret = __curtiered_position_tier(curtiered, c, true, &cmp)) == 0 && cmp == 0 &&
               curtiered->current == NULL)
                 curtiered->current = c;
             WT_ERR_NOTFOUND_OK(ret, false);
         }
-        F_SET(curtiered, WT_CLSM_ITERATE_NEXT | WT_CLSM_MULTIPLE);
-        F_CLR(curtiered, WT_CLSM_ITERATE_PREV);
+        F_SET(curtiered, WT_CURTIERED_ITERATE_NEXT | WT_CURTIERED_MULTIPLE);
```

Comment:
I know you just did a rote replacement,  but please alphabetize these (and please feel free to fix LSM where it is almost certainly wrong/unalphabetized too).

Reply:
Do you mean the flags on either side of the OR?  They're already alphabetized.  Or am I missing something else here?

Reply:
You are right! My eyes only saw the tail of the flag name `_NEXT` versus `_MULTIPLE`. Never mind!